pkg/ingester/ingester.go

```diff
@@ -231,6 +234,8 @@ func (cfg *Config) RegisterFlags(f *flag.FlagSet, logger log.Logger) {
 	f.BoolVar(&cfg.UpdateIngesterOwnedSeries, "ingester.track-ingester-owned-series", false, "This option enables tracking of ingester-owned series based on ring state, even if -ingester.use-ingester-owned-series-for-limits is disabled.")
 	f.DurationVar(&cfg.OwnedSeriesUpdateInterval, "ingester.owned-series-update-interval", 15*time.Second, "How often to check for ring changes and possibly recompute owned series as a result of detected change.")
 
+	f.BoolVar(&cfg.EnablePushAPI, "ingester.enable-push-api", true, "When disabled, disallows Push API in ingester. Use after migrating ingester to ingest-storage.")
```

Comment:
I'm not too fancy about the phrasing here, and am open for better options, that would sound `help`-ful :)

Reply:
Maybe something like this?

```suggestion
	f.BoolVar(&cfg.EnablePushAPI, "ingester.enable-push-api", true, "Enables Push gRPC method on ingester. This can only be disabled when using ingest-storage to make sure that ingesters only receive data from Kafka.")
```